src/services.js

```diff
@@ -610,7 +610,6 @@ export class Services {
    * @return {!./service/timer-impl.Timer}
    */
   static timerFor(window) {
-    // TODO(alabiaga): This will always return the top window's Timer service.
```

Comment:
Need to change `getService` to `getExistingServiceForDocInEmbedScope`. 

First line of `getService` is `win = getTopWindow(win)`.

Reply:
Do you mean it works incorrectly now?

Reply:
Yea, that's what the TODO meant.

Reply:
I see. Do you want me to change this here? Would be happy to, but that could have deeper ripples...

Reply:
I reverted this change for now - too many ripple effects to fight through in a preliminary PR.